features/bootstrap/FunctionsContext.php

```diff
@@ -90,7 +90,7 @@ public function iExpectWhenIRunWithArgs( $return, $function, TableNode $args ) {
 	/**
 	 * @Then I expect :return when I run :function
 	 */
-	public function iExcpectWhenIRun( $return, $function ) {
+	public function iExpectWhenIRun( $return, $function ) {
```

Comment:
Did you fully check the implications of renaming this method? I know it is a typo, but it is considered code change, what do you think about defining a new alias method to it, and marking this one as deprecated? ex:
```php
	 /**
	  * @Then I expect :return when I run :function
	  *
	  * @deprected use static::iExpectWhenIRun instead
	  */
	public function iExcpectWhenIRun( $return, $function ) {
		static::iExpectWhenIRun( $return, $function )
	}

	 /**
	  * @Then I expect :return when I run :function
	  */
	public function iExpectWhenIRun( $return, $function ) {
		$this->iExpectWhenIRunWithArgs( $return, $function, new TableNode( array( array() ) ) );
	}
```

Reply:
Done.
TBH I treat every thing as text when I fix typos.

Reply:
No worries, it happens, thanks for the fixes

Reply:
Glad to contribute.